frame/staking/src/lib.rs

```diff
@@ -2288,9 +2294,10 @@ pub mod pallet {
 		///
 		/// The dispatch origin for this call must be _Signed_, but can be called by anyone.
 		///
-		/// If the caller is the same as the controller being targeted, then no further checks
-		/// are enforced. However, this call can also be made by an third party user who witnesses
-		/// that this controller does not satisfy the minimum bond requirements to be in their role.
+		/// If the caller is the same as the controller being targeted, then no further checks are
+		/// enforced. However, this call can also be made by an third party user who witnesses that
+		/// the limits for number of validators or nominators is near the threshold (75%), and that
```

Comment:
Once the 75 is a named value, this should refer to the value instead of the literal.